UserScrape/app.py

```diff
@@ -4,21 +4,32 @@
 from selenium.common.exceptions import NoSuchElementException
 from data import load_all_seeds, seeds_for_user
 import asyncio
-import random
+import argparse
+from typing import List
 
+def experiment(initialization: bool, accounts: List[int]):
+    cfg = load_cfg()
+    video_seeds_df = load_all_seeds()
 
-cfg = load_cfg()
-video_seeds_df = load_all_seeds()
-repetitions = 2 # 100
-# todo: this list of videos needs to be sampled
-test_videos = ['uo9dAIQR3g8', 'CH50zuS8DD0', '9_R3_CThc38']
 
-for user in cfg.users:
-    print(f'scraping for user {user.email}')
-    crawler = Crawler(cfg.data_storage_cs, user.email, user.password, user.telephone_number, cfg.headless)
-    crawler.test_ip()
 
-    user_seed_videos = seeds_for_user(user, video_seeds_df)
+    cfg = load_cfg()
+    if initialization:
+        video_seeds_df = load_all_seeds()
+    else:
+        # todo: function that returns only 5 sampled videos
+        video_seeds_df = load_all_seeds()[0:5]
```

Comment:
Do we need a different function for this or can we add an argument to the old one?

Reply:
an argument to the old one I think

Reply:
actually, probably separate functions (1st coffee). Going to work on getting the list for experiment runs, is this ready to check in?